### ISSUE TITLE
`vue/valid-v-for` should not trigger on looping through only values of an object

### ISSUE THREAD

**Issue by aibolit** [vuejs/eslint-plugin-vue#2733]
**What rule do you want to change?**
[vue/valid-v-for](https://eslint.vuejs.org/rules/valid-v-for.html)

**Does this change cause the rule to produce more or fewer warnings?**
fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**
new default behavior

**Please provide some example code that this change will affect:**

```vue
  v-for="(, foo) in bar"
```

**What does the rule currently do for this code?**
`error  Invalid alias ''  vue/valid-v-for`

**What will the rule do after it's changed?**
not error out on looping through only keys of an Object

**Additional context**

**Comment by waynzh** [comment]
Sorry for the late reply. This is indeed valid syntax, but in my personal opinion, the `invalidEmptyAlias` error was intentionally added to help prevent potential issues.

You can avoid this report by explicitly enumerating all variables, such as using something like `v-for="(_, foo) in bar"`. This makes the intent clearer and avoids ambiguity.

The existing PR could implement an `allowEmptyAlias` option (defaulting to `false`). Once implemented, users could enable it by setting it to `true` to bypass this check.
